ext/node/polyfills/process.ts

```diff
@@ -705,7 +706,7 @@ internals.__bootstrapNodeProcess = function (
   Object.defineProperty(argv, "1", {
     get: () => {
       if (Deno.mainModule.startsWith("file:")) {
-        return fromFileUrl(Deno.mainModule);
+        return pathFromURL(Deno.mainModule);
```

Comment:
`Deno.mainModule` is a string url, and `pathFromURL` only works for `URL` input (unlike `fromFileUrl`). I think it needs to be converted to URL first

```suggestion
        return pathFromURL(new URL(Deno.mainModule));
```

Reply:
Oh! thanks. I'll update. I really appreciate the help :)

Reply:
Thanks for finding this bug. I was so confused why the build was failing in CI when this was working on my local machine lol (using the provided devcontainer).

This brings up the question on why the failing test would pass on my local machine but fail in CI despite the failed step also using a posix OS. 🤔